modules/ROOT/pages/anypoint-monitoring-visualizer.adoc

```diff
@@ -0,0 +1,180 @@
+|Processing mule runtime events for Visualizer
+|ui
+|Nginx service for serving static UI artifact files
+
+|===
+
+=== DIAS Components
+
+DIAS provides the foundations for data transmission between the Mule runtime (running Filebeat) and the InfluxDB storing that data. Once data reaches InfluxDB, it becomes available to the Anypoint Monitoring UI as well as the Insight alerting engine.
+
+The `dias-prov-k8s-amr-ingestor-router-comp` pods run Nginx accepting connections from remote runtimes thought mTLS (it uses a node-port service). Data is then forwarded to `dias-prov-k8s-am-ingestor-comp` pods running Logstash.
+
+Logstash processes and transform those messages and batch them before hitting InfluxDB. The `dias-prov-k8s-am-influxdb-comp` component is a stateful InfluxDB cluster consisting in 3 meta and 2 data nodes.
+
+The Insight alerting engine is managed by the `dias-prov-k8s-insight-comp` and the `dias-prov-k8s-cassandra-comp` component. Both of these components are statefulsets. It periodically fetches metrics from InfluxDB and based on the customer's alerts definition, the engine triggers alarms. Each pod in `dias-prov-k8s-insight-comp` has 3 main containers, including a Mule Runtime, Meld and Meld-Drill.
+
+Logstash, Insight and InfluxDB are all K8s stateful sets.
+
+
+=== Monitoring Center Components
+
+Monitoring Center components represents the "Anypoint Monitoring" product that visualizer the data stored in DIAS products.
+
+All user traffic goes to the experience api (`monitoring-center-ui-api`) and from there it can proxied to:
+
+- `monitoring-center-settings-api` to manage user settings.
+
+- `monitoring-center-alerts-api` to manage alerts, synchronize the changes with Insights engine and trigger basic alerts notifications.
+
+- `monitoring-center-visualizer` that is Grafana fork. It manages the dashboards logic, advanced alerts and the rest of the functionality available in PCE 3.0.
+
+- `monitoring-center-metrics-api` to validate Influx DB queries and proxy the calls to the data source (`dias-prov-k8s-am-influxdb-comp`).
+
+=== Visualizer Components
+
+Visualizer components represents the "Anypoint Visualizer" product.
+
+Logstash `dias-prov-k8s-am-ingestor-comp` sends a subset of mule agent events to the `visualizer-topology-processor` service.
+That service process those events and stores the results in the Postgres DB (`stolon-amv`).
+The final application network generation will be handled by `visualizer-topology-api` service and is invoked by a user traffic.
+There is a set of cron jobs to prevent the uncontrolled DB growth for mule agent events. They perform the stale data clean and runs as part of `visualizer-janitor` service.
+
+All user traffic comes to the `visualizer-experience-api` service that orchestrates loading all necessary data from different data sources (like `visualizer-topology-api`, `monitoring-center-ui-api`, etc.) and performs further data decorations.
+
+== Considerations
+
+=== Cluster secrets
```

Comment:
This also should be moved to the runbook. We don't want implementation details in the public docs

Reply:
Done